mava/components/jax/executing/action_selection.py

```diff
@@ -34,14 +32,29 @@ class ExecutorSelectActionProcessConfig:
 
 class ExecutorSelectAction(Component):
     @abc.abstractmethod
-    def __init__(self, config: Any) -> None:
+    def __init__(
+        self,
+        config: ExecutorSelectActionProcessConfig = ExecutorSelectActionProcessConfig(),
```

Comment:
Also minor, but isn't it more intuitive to have it called `ExecutorSelectActionConfig` to match the component name?

Reply:
Also agreed, changed